src/DataProtection/Azure.KeyVault.Keys/src/Microsoft.AspNetCore.DataProtection.Azure.KeyVault.Keys.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Microsoft Azure KeyVault key encryption support.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageTags>aspnetcore;dataprotection;azure;keyvault</PackageTags>
+    <IsShippingPackage>true</IsShippingPackage>
+    <HasReferenceAssembly>false</HasReferenceAssembly>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.AspNetCore.DataProtection.Azure.KeyVault.Keys.Tests" />
```

Comment:
Suggest using one mechanism for ITVs in this project. If you choose to stick w/ this element, it takes a `Key` attribute for use in the `DynamicProxyGenAssembly2` case.